ui/src/components/ProvisionD-Config.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <TabMenu :model="TabMenuItem" :activeIndex="activeIndex"/>
+    <router-view></router-view>
+</template>
+<script setup lang="ts">
+
+import TabMenu from 'primevue/tabmenu';
+import { computed, ref } from 'vue';
+
+const activeIndex = ref(0);
```

Comment:
Looks like the `activeIndex` is used properly here, but as you mentioned, doesn't work. I don't know why. It's possible this is an issue with this version of PrimeVue. Routing straight to `/provisionConfig/reqDefinition` like you have it seems fine to me though, and this line can be removed.

If you really want the main `/provisionConfig` route to render the ReqDefForm component, however, you could change the routing below to just `/provisionConfig` and the main index routing to use `path: ''` for the first child route.

Reply:
Done with changes